demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -190,6 +190,35 @@ def test_is_changed_command_name_or_arg(self, current, old, answer):
         structure.quiet_bc = True
         assert validator.is_changed_command_name_or_arg() is False  # if quiet_bc is true should always succeed
 
+    CHANGED_COMMAND_OR_ARG_MST_TEST_INPUTS = [
+        ([{"name": "test", "arguments": [{"name": "test1"}]}, {"name": "test2", "arguments": [{"name": "test3"}]}],
+         [{"name": "test", "arguments": [{"name": "test"}, {"name": "test1", "required": True}]},
+         {"name": "test2", "arguments": [{"name": "test2"}, {"name": "test3", "required": True}]}],
+         "[ERROR]: : [BC104] - Possible backwards compatibility break, You've changed the name of some of the command(s) or its arg in the file,"
+         " please undo, the following commands:\ntest\ntest2")
+    ]
+
+    @pytest.mark.parametrize("current, old, expected_error_msg", CHANGED_COMMAND_OR_ARG_MST_TEST_INPUTS)
+    def test_is_changed_command_name_or_arg_msg(self, capsys, current, old, expected_error_msg):
```

Comment:
Seems to me you are doing a couple of BC changes here (you have removed arguments and removed the required field), let's make it simple or add two different cases.